kotlin/display-grid/README.md

```diff
@@ -0,0 +1,36 @@
+# Display grid
+
+Display coordinate system grids including Latitude/Longitude, MGRS, UTM and USNG on a map view. Also, toggle label visibility and change the color of grid lines and grid labels.
+
+![Image of display grid](display-grid.png)
+
+## Use case
+
+Grids are often used on printed maps, but can also be helpful on digital maps, to identify locations on a map.
+
+## How to use the sample
+
+Tap on the `Change Grid` button in the toolbar to open a settings view. You can select type of grid from `Grid Type` (LatLong, MGRS, UTM and USNG) and modify its properties like label visibility, grid line color, and grid label color.
+
+## How it works
+
+1. Create an instance of one of the `Grid` types.
```

Comment:
since the PR's open, let's number these 1, 2, 3 etc in keeping with the changes in the common readme repo.

Reply:
No problem!